js/src/forum/components/PostStream.js

```diff
@@ -349,7 +346,7 @@ export default class PostStream extends Component {
       // We manually set the index because we want to display the index of the
       // exact post we've scrolled to, not just that of the first post within viewport.
       this.updateScrubber();
-      this.stream.index = index;
+      this.stream.index = index || this.stream.index;
```

Comment:
Shouldn't it be `index + 1`? 
```suggestion
     if (index !== undefined) this.stream.index = index + 1;
```
We input the post index (which can be 0), but we use this.stream.index as the scrubber index in other parts of the code (which is post index + 1).